### DISCUSSION TITLE
Send-As in GMail?

### DISCUSSION THREAD

**Discussion by flancast90** [Miksus/red-mail#41] [Q&A]
Hi!

I use gmail to link/send from my business account, which is enabled via Gmail "Add an Account" feature in Settings. Basically, this means that I can choose an account to send from and the name changes from my personal account to business account. Since I do not have a separate login for the business account (because of how gmail handles the name-change at send), I cannot send from my business account using redmail.

Is there perhaps a way not mentioned in the docs, or is this not available as of right now?

**Comment by Miksus** [comment]
Hi!

Basically what Red Mail only does is that it just communicates SMTP servers (using smtplib). My understanding (which could be incorrect) is that for example, the "Add an account" in Outlook/Gmail means that it will create read access to the account using possibly POP/IMAP and then the send functionality is linked using SMTP (which Red Mail is also capable of). I suspect Gmail/Outlook has the information of the hosts and ports of common email providers (Gmail & Outlook found [here](https://red-mail.readthedocs.io/en/latest/tutorials/config.html)) and they just map it perhaps depending on the domain to make this linking convenient. 

Is there a way you to get the host and port of the SMTP server of the business email provider? Commonly, staff point/IT support should know or be able to find out the host and port of the SMTP server. Also ask whether it requires login as sometimes passing credentials may not be needed if already inside the company's network.

Or are you able to access this information?

A disclaimer: I'm not sure how great this is to be advertised but my knowledge about SMTP is based on trying what just works.

**Reply by flancast90** [reply]
Nevermind, I got it working! For future users with the same issue, I just used the gmail example from the docs, and added:

```
...
sender="diff. email",
...
```

It seems pretty self explanatory when you see it now, but there seemed to be nothing in the docs...

**Reply by Miksus** [reply]
Great! Actually, I tried to explain the sender [in Sending Emails](https://red-mail.readthedocs.io/en/latest/tutorials/sending.html#sending-emails). Maybe the wording could be changed to be more obvious though.
